modular_zubbers/code/modules/hydroponics/code/grown/rocks.dm

```diff
@@ -91,3 +92,421 @@
 
 /datum/reagent/consumable/nutriment/mineral
 	taste_description = "rocks and stones"
+
+
+//Rockfruits evolutions? OREFRUITS!//
+//Sand - Base tier breaks into 4 trees ('energy', Precious, Metal, Miscmats)
+/obj/item/seeds/sandfruit
```

Comment:
Since none of these are setting plant stats, won't they simply have the default ones? They should inherit from /obj/item/seeds/rockfruit by being a subtype of it, that's how the other plant mutations handle it

Reply:
I wanted them to be a weaker version, meaning that they can die due to being less sturdy and require more attention.

Reply:
This means that all of the orefruit will have the stats: lifespan = 25, endurance = 15, yield = 3, potency = 10, so they'll have more lifespan and yield? It would probably make sense if the yield was more limited given they let you have ore.

Reply:
Yield is capped by the fact it has a trait that caps it at five and cools it down (to prevent plasma vaporizing)